### DISCUSSION TITLE
What happened to issues on github for this api?

### DISCUSSION THREAD

**Discussion by ghankerson** [weather-gov/api#423] [General]
In particular there was an issue we were tracking about the api serving stale data that was here: https://github.com/weather-gov/api/issues/71. Now there are no issues - what happened?

**Comment by scadergit** [comment]
All issues were converted to the new discussions feature, given the actual code base is not on GitHub, and the goal of this forum was meant more towards how to use the API and not operational issues, which should be sent to the helpdesk.

**Reply by ghankerson** [reply]
Thank you.  Is there a place to track previous tickets?

**Reply by StephenClouse** [reply]
All of the previous issues were converted to discussions, so they're all here _somewhere_. Unfortunately it doesn't seem like GitHub maintains the link between them after conversion.

Issue 71, for example, became [discussion 329](https://github.com/weather-gov/api/discussions/329), apparently.
